website/src/app/[lang]/[region]/(website)/campaign/[campaign]/page.tsx

```diff
@@ -47,127 +65,177 @@ export default async function Page({ params }: CampaignPageProps) {
+							<div>
+								<Progress value={percentageCollected} className={'h-6'} />
+							</div>
+							<div className="flex">
+								<div className="flex-1 text-left">
+									<Typography size="md" color="primary">
 										{translator.t('campaign.with-goal.collected-amount', {
 											context: {
 												count: contributions,
 												amount: amountCollected,
 												currency: campaign.goal_currency,
 											},
 										})}
-									</div>
-									<div className=" text-primary w-1/2 text-right">
+									</Typography>
+								</div>
+								<div className="flex-1 text-right">
+									<Typography size="md" color="primary">
 										{translator.t('campaign.with-goal.goal-amount', {
 											context: {
 												amount: campaign.goal,
 												currency: campaign.goal_currency,
 											},
 										})}
-									</div>
+									</Typography>
 								</div>
 							</div>
-						)}
-					</div>
-					{daysLeft >= 0 && (
-						<>
-							<div className="flex flex-col items-center">
-								<Typography size="xl" weight="bold" color="accent">
-									{translator?.t('campaign.days-left', { context: { count: daysLeft } })}
-								</Typography>
-							</div>
-							<div className="mt-4 w-full">
-								<OneTimeDonationForm
-									lang={params.lang}
-									region={params.region}
-									translations={{
-										oneTime: translator.t('donation-interval.0.title'),
-										monthly: translator.t('donation-interval.1.title'),
-										amount: translator.t('amount'),
-										submit: translator.t('button-text'),
-									}}
-									campaignId={params.campaign}
-								/>
+						</div>
+						<div className="flex items-center justify-center" style={{ height: '500px' }}>
+							<div className="card bg-primary w-full rounded-xl p-6">
+								<div>
+									<Typography size="xl" color="popover">
+										{translator.t('campaign.card-title')}
+									</Typography>
+								</div>
+								<div className="mt-3">
+									<OneTimeDonationForm
+										lang={params.lang}
+										region={params.region}
+										translations={{
+											oneTime: translator.t('donation-interval.0.title'),
+											monthly: translator.t('donation-interval.1.title'),
+											amount: translator.t('amount'),
+											submit: translator.t('button-text-short'),
+										}}
+										campaignId={params.campaign}
+									/>
+								</div>
+								<div className="mt-4 text-center">
+									<Typography size="md" color="popover">
+										{translator?.t('campaign.days-left', { context: { count: daysLeft } })}
+									</Typography>
+								</div>
 							</div>
-						</>
-					)}
-					{daysLeft < 0 && (
-						<div className="flex flex-col items-center">
-							<Typography size="xl" weight="bold" color="accent">
-								{translator?.t('campaign.ended', { context: { count: daysLeft } })}
+						</div>
+					</div>
+				</div>
+			</BaseContainer>
+			<BaseContainer>
```

Comment:
Let's only render this when the optional campaign.second_description and campaign.third_description are defined. 

Let's wrap it again with `{campaign.second_description && campaign.third_description && (`?